django/db/backends/oracle/base.py

```diff
@@ -241,7 +241,7 @@ def init_connection_state(self):
         if not self.get_autocommit():
             self.commit()
 
-    def create_cursor(self, name=None):
+    def create_cursor(self, name=None, *args, **kwargs):
         return FormatStylePlaceholderCursor(self.connection)
```

Comment:
IMO you should also pass args/kwargs to `FormatStylePlaceholderCursor.__init__` and change `FormatStylePlaceholderCursor` initialization e.g.:
```diff
diff --git a/django/db/backends/oracle/base.py b/django/db/backends/oracle/base.py
index 29ae2a6..8c1fe35 100644
--- a/django/db/backends/oracle/base.py
+++ b/django/db/backends/oracle/base.py
@@ -383,8 +383,8 @@ class FormatStylePlaceholderCursor:
     """
     charset = 'utf-8'
 
-    def __init__(self, connection):
-        self.cursor = connection.cursor()
+    def __init__(self, connection, *args, **kwargs):
+        self.cursor = connection.cursor(*args, **kwargs)
         # Necessary to retrieve decimal values without rounding error.
         self.cursor.numbersAsStrings = True
```